test/e2e/e2e_suite_test.go

```diff
@@ -174,6 +182,45 @@ var _ = BeforeSuite(func() {
 		ClientSet:        kubernetes.NewForConfigOrDie(managedConfig),
 		ControllerName:   "status-sync-controller-test",
 	}
+
+	if !gkSyncDisabled {
```

Comment:
So this shuts down the controller and restarts it even before the tests are run? I guess that works... 😅

Reply:
Yeah, because there were two `Describes` that seemed to test some Gatekeeper stuff, I decided to do the fake uninstall and reinstall in the BeforeSuite, when I know that process won't interfere with any running tests.